app/src/pages/inside/filtersPage/filtersPage.jsx

```diff
@@ -148,6 +161,24 @@ export class FiltersPage extends Component {
     this.props.tracking.trackEvent(FILTERS_PAGE_EVENTS.CLICK_ADD_FILTER_BTN);
   };
 
+  renderNoItemMessage = () => (
```

Comment:
I see it duplicates in `membersPage.jsx`.
Could we create one common component f.e. `noFiltersFound` and hold it there?

Reply:
fixed